airbyte-workers/src/main/java/io/airbyte/workers/normalization/DefaultNormalizationRunner.java

```diff
@@ -43,7 +43,7 @@
 
   private static final Logger LOGGER = LoggerFactory.getLogger(DefaultNormalizationRunner.class);
 
-  public static final String NORMALIZATION_IMAGE_NAME = "airbyte/normalization:0.1.0";
+  public static final String NORMALIZATION_IMAGE_NAME = "airbyte/normalization:dev";
```

Comment:
can you create an issue with the proper milestone so we fix that before releasing?